docs/providers.rst

```diff
@@ -415,6 +415,7 @@ Eth
 - :meth:`web3.eth.send_transaction() <web3.eth.Eth.send_transaction>`
 - :meth:`web3.eth.send_raw_transaction() <web3.eth.Eth.send_raw_transaction>`
 - :meth:`web3.eth.hashrate <web3.eth.Eth.hashrate>`
+- :meth:`web3.eth.chain_id <web3.eth.Eth.chain_id>`
```

Comment:
nit: looks like this list is alphabetized, if you dont mind moving `chain_id` and `hashrate` into appropriate spots.